Makefile

```diff
@@ -23,7 +23,7 @@ $(plugin_path): $(uberjar_path)
 	cd target/plugin; jar xf ../../$(uberjar_path)
 	find target/plugin -type f -path 'target/plugin/clojure/repl*' -delete
 	find target/plugin -type d -empty -delete
-	cd target/plugin; jar cmf META-INF/MANIFEST.MF $(plugin_name) plugin.xml amperity cheshire clj_http clojure com org potemkin riddley slingshot vault *.class *.clj *.java *.xml
+	cd target/plugin; jar cmf META-INF/MANIFEST.MF $(plugin_name) plugin.xml amperity cheshire clj_http clojure com environ envoy org potemkin riddley slingshot vault *.class *.clj *.java *.xml
```

Comment:
A note for another time: it might be worth investigating if this whole whitelisting process is necessary like it is in the other plugin project.

Reply:
Definitely agree. I'm going to look into that, because this bit has been a pain to handle so far!